undertow/src/main/resources/org/wildfly/extension/undertow/LocalDescriptions.properties

```diff
@@ -319,6 +319,7 @@ undertow.setting.session-cookie.http-only=Is cookie http-only
 undertow.setting.session-cookie.secure=Is cookie secure?
 undertow.setting.session-cookie.domain=Cookie domain
 undertow.setting.session-cookie.comment=Cookie comment
+undertow.setting.session-cookie.comment.deprecated=Cookie comments were deprecated for removal in version 6.0 of the Jakarta Servlet specification.
```

Comment:
Let's be more descriptive here and save users some time and explain (1) the RFC part of this (i.e. RFC 6265) and (2) that IIUC this is not a typical deprecation, since it actually has no effect (`jakarta.servlet.http.Cookie#setComment` is simply a no-op). Something like:

> Cookie comments were deprecated for removal in version 6.0 of the Jakarta Servlet specification as they are no longer required with RFC 6265 and the value is ignored.

Reply:
Done.